src/ndsctl_thread.c

```diff
@@ -313,6 +318,17 @@ ndsctl_auth(FILE *fp, char *arg)
 	}
 	UNLOCK_CLIENT_LIST();
 
+	/* set client values */
+	client->download_limit = download;
```

Comment:
Why not assign zero here instead of using a dummy variable?
Same for upload and now.

Reply:
The intention is to integrate binauth functionally as a next enhancement. The values of these variables will be set by the binauth script.
I will add a TODO comment here for now.